src/components/App.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import Display from './Display';
+import ButtonPanel from './ButtonPanel';
+
+function App() {
+  return (
+    <div className="calculator-body">
+      <Display result="0" />
```

Comment:
Since the `Display` component already has `0` as the `defaultProps`, there is no need to add this anymore unless you are passing a value different from `0`.
```suggestion
      <Display />
```